server/src/query.rs

```diff
@@ -81,12 +81,26 @@ impl Query {
         let runtime_config = runtime_config.with_memory_limit(pool_size, fraction);
         let runtime = Arc::new(RuntimeEnv::new(runtime_config).unwrap());
 
-        let config = SessionConfig::default()
+        let mut config = SessionConfig::default()
             .with_parquet_pruning(true)
             .with_prefer_existing_sort(true)
             .with_round_robin_repartition(true);
 
-        let state = SessionState::new_with_config_rt(config, runtime);
+        config.options_mut().execution.parquet.enable_page_index = true;
+        config.options_mut().execution.parquet.pushdown_filters = true;
+        config.options_mut().execution.parquet.reorder_filters = true;
+        config
+            .options_mut()
+            .execution
+            .parquet
+            .schema_force_view_types = true;
```

Comment:
Also, let's add comments here on what are these options and why it makes sense to enable these.